scripts/memfault/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _mds_ble_gateway_script:
+
+MDS BLE gateway script
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The MDS BLE gateway is an application used as a client for the :ref:`mds_readme`.
+It forwards data received on the Memfault Diagnostic service to the Memfault`s cloud for further analysis.
+The destination data URI and the Memfault Project Key are read from service characteristics.
+
+Overview
+********
+
+The scripts scans for the :ref:`mds_readme` service UUID in advertising packets.
+Once it founds the MDS service UUID, it connect to the remote device and reads MDS data.
+Next the scripts waits for MDS notifications and forwards it to the URI read from the URI characteristics.
+
+For more details about the Memfault check `Memfault SDK`_ and the :ref:`mod_memfault` to check details of the integration with |NCS|.
+
+Requirements
+************
+
+The script source files are located in the :file:`scripts/memfault` directory.
+
+To install the script's requirements, run the following command in its directory:
+
+.. code-block:: console
+
+   python3 -m pip install -r requirements.txt
+
+This script also required the nRF52 series development kit to run the connectivity firmware that is programmed automatically when running the script.
+
+Using the script
+****************
+
+Connect the nRF52 series development kit to your PC.
+Run the script commands in the :file:`scripts/memfault` directory to start using the script.
+Your DK will be programmed with connectivity firmware.
+To list the script arguments, run the following command:
+
+.. code-block:: console
+
+   python3 mds_ble_gateway.py -h
+
+The following help information describes the available script arguments:
+
+.. code-block:: console
+
+   Memfault BLE gateway
+
+   optional arguments:
+     -h, --help            show this help message and exit
+     --snr SNR             Segger chip id
+     --com COM             COM port name. For example COM0 or /dev/ttyACM0
+     --erase, -e           Erase target device before flashing firmware
+     --bond-disable        Disable bonding simulation
+     --reconnections RECONNECTIONS
+                           Number of reconnection attempts
+     --timeout TIMEOUT, -t TIMEOUT
+                           Connection establish timeout in seconds
```

Comment:
```suggestion
   optional arguments:
     -h, --help             Show this help message and exit
     --snr SNR             Segger chip ID
     --com COM             COM port name. For example COM0 or /dev/ttyACM0
     --erase, -e           Erase target device before flashing the firmware
     --bond-disable        Disable bonding simulation
     --reconnections RECONNECTIONS
                           Number of reconnection attempts
     --timeout TIMEOUT, -t TIMEOUT
                           Connection establish timeout in seconds
```
If these are already in the script help and you will not fix them in there, leave them as they are

Reply:
I can fix almost everything apart from `help` which is build-in